### ISSUE TITLE
[Bug]: The IaC results use absolute paths 

### ISSUE THREAD

**Issue by lukaspj** [advanced-security/codeql-extractor-iac#138]
### What happened?

The results from this IaC extension has absolute paths somewhere, which means that GitHub can't link issues to the file.
The results from clicking the "view file" button here:
![image](https://github.com/advanced-security/codeql-extractor-iac/assets/2369614/0b04edc6-9a61-43e3-92f5-66f3c4490113)
Goes to:
https://github.com/LEGO/novus-api/blob/main//home/runner/work/<repo>/<path>.Dockerfile#L51

### Version

0.1.0

**Comment by GeekMasher** [comment]
Thanks for raising this @lukaspj, sad I am away the rest of the week and next. Does this urgently need fixing?

**Comment by lukaspj** [comment]
Nope, it only applies to what it considers as syntax errors, so it doesn’t affect all our repositories and the syntax error should be fixable if we can figure out what it doesn’t like

**Comment by GeekMasher** [comment]
When you say "syntax errors", is that in the Dockerfile or something else? Maybe there is an issue with the parser / analysis

**Comment by lukaspj** [comment]
Well I only know what’s in the screenshot 😅

but it’s only an issue in a single repo
